.github/workflows/test-pull-request.yml

```diff
@@ -16,45 +19,80 @@ on:
   pull_request:
     branches: [main]
 
+###############
+# Set the Job #
+###############
 jobs:
-  validate:
+  lint:
```

Comment:
I prefer "validate" because we're doing more than just linting (e.g., checking links, running a test build). We'll probably add more validation in the future.

Reply:
Sounds good to me.